mail/mail_test.go

```diff
@@ -0,0 +1,105 @@
+package mail
+
+import (
+	"testing"
+)
+
+func TestIpChangedTemplate(t *testing.T) {
+	model := struct {
+		IpAddress string
+	}{
+		IpAddress: "127.0.0.1",
+	}
+	content, err := generateEmailContent("../templates/ip_changed.html.tmpl", model)
+	if err != nil {
+		t.Fatalf("Failed to generate content. %v", err)
+	}
+	if len(content) == 0 {
+		t.Fatalf("Failed to generate content.  Result is empty")
+	}
+}
+
+func TestAccessRevokedLeadershipTemplate(t *testing.T) {
+	model := struct {
+		Name  string
+		Email string
+	}{
+		Name:  "Member Name",
+		Email: "member@email.com",
+	}
+	content, err := generateEmailContent("../templates/access_revoked_leadership.html.tmpl", model)
+	if err != nil {
+		t.Fatalf("Failed to generate content. %v", err)
+	}
+	if len(content) == 0 {
+		t.Fatalf("Failed to generate content.  Result is empty")
+	}
+}
+
+func TestAccessRevokedTemplate(t *testing.T) {
+	model := struct {
```

Comment:
Noticed you used the exact same model multiple times in this file. Would it be alright to make it reusable and have it be created once?

Reply:
Thanks for the feedback.  Refactored